sdk/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import typescript from "rollup-plugin-typescript2";
```

Comment:
Would this ever effect this library being embedded with bundlers like WebPack or Vite?

Reply:
No, it's only used while building, the compiled code should work identically in every bundler (Webpack, Snowpack, Rollup, Parcel, etc.)